docs/audit_report/src/00_09_introduction.rst

```diff
@@ -207,10 +202,11 @@ reviewed:
 The following previously existing modules are now in scope
 and were fully reviewed:
 
-- argon2, argon2_avx2, argon2_ssse3
-- blake2
-- hkdf
-- shake
+- shake_xof
```

Comment:
Maybe make it clear in the TODO that e.g. this new `shake_xof` should also be added to the list above of all modules.